sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1159,7 +1167,6 @@ public void close() throws IOException {
     @Nullable abstract String getTopic();
     @Nullable abstract Coder<K> getKeyCoder();
     @Nullable abstract Coder<V> getValueCoder();
-    abstract boolean getValueOnly();
```

Comment:
This and the changes below don't seem to be related to the current commit or PR. Also, all of these changes are missing tests.